config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get '/' => 'pages#home'
```

Comment:
I think this might be older syntax for how to set up routes in rails. Might be wrong though as I'm no expert in rails myself. Generally the way I've seen it done in Portal is like this `get '/', to: 'pages#home'`.

Reply:
Actually, if you look at the rails documentation [here](https://guides.rubyonrails.org/routing.html#using-root) it looks like it can simply be done with `root 'pages#home'`

Reply:
Thanks James! I was adapting my code from an example which was pointing to a particular path. Great to find out there is this elegant solution for setting the root.